crates/steel/src/lib.rs

```diff
@@ -203,6 +229,12 @@ pub trait EvmBlockHeader: Sealable {
     fn timestamp(&self) -> u64;
     /// Returns the state root hash.
     fn state_root(&self) -> &B256;
+    #[cfg(feature = "unstable-event")]
```

Comment:
This adds new fields we require from a block. This is a breaking change of the EvmBlockHeader trait.
And, unfortunately, it is difficult to hide this change behind feature flags.

Reply:
I think this is probably ok. I would be a bit surprised if anyone is implementing this themselves right now, and if anyone screams we can roll back the release and bump to 2.0 when we release this.